src/external-scripts/inkeep.ts

```diff
@@ -47,9 +47,9 @@ const aiChatSettings = {
       },
     ],
     quickQuestions: [
-      'How do I get presence data for a specific member?',
-      "What's the difference between detach and unsubscribe?",
-      "Can I limit users' access to message interactions?",
+      'What is a channel?',
+      "How do I authenticate with Ably?",
+      "How to manage user status?",
```

Comment:
Small thing: but I'd keep this consistent with the authentication question and make it 'How do I manage user status?'

Reply:
Fixed in c16bb8ca71e382531572b4724171be429d7e26a5

Reply:
Got a consistent linting error. Shout if you want me to take a look 🙂

Reply:
No it is my bad :( sorry, was rushing yesterday with this.